src/main/java/uk/gov/hmcts/ccd/domain/types/ValidationContext.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.ccd.domain.types;
```

Comment:
It would be good trying to add this class for all custom validators which extends from DataFieldIdValidator or  PredefinedTypeFieldValidator .  That would lead to deleting  DataFieldIdValidator class. 

ValidationContext should be available for all validators  types. regardless its data us use or not.